packages/airnode-validator/package.json

```diff
@@ -28,6 +28,7 @@
   "devDependencies": {
     "@types/yargs": "^17.0.2",
     "copyfiles": "^2.4.1",
+    "lodash": "^4.17.21",
```

Comment:
Why is this here?

Reply:
that's my bad commiting, it belongs to another branch, removed in 9f1ef8c